lisa/tools/python.py

```diff
@@ -89,3 +94,74 @@ def exists_package(self, package_name: str) -> bool:
     def uninstall_package(self, package_name: str) -> bool:
         result = self.run(f"uninstall {package_name} -y", force_run=True, sudo=True)
         return result.exit_code == 0
+
+
+class PythonVenv(Tool):
+    @property
+    def command(self) -> str:
+        path = self.get_venv_path().joinpath(f"bin/{self._python.command}")
```

Comment:
Because get_venv_path returns a purepath already, and it's for this kind of scenario.

```python
return self.get_venv_path() / "bin" / self._python.command
```

Reply:
updated